core/controllers/editor_test.py

```diff
@@ -554,7 +554,7 @@ def test_exploration_download_handler_for_default_exploration(self):
 
         # Check download to JSON.
         exploration.update_objective('Test JSON download')
-        exp_services._save_exploration(  # pylint: disable=protected-access
+        exp_services.update_exploration(
```

Comment:
@aggarwalpulkit596, have you tested these changes? You can run a specific backend test using `--test_target`` flag, see [this wiki page](https://github.com/oppia/oppia/wiki/Running-Tests#server-side-tests) for more info.)

Reply:
Will add a screenshot